gax-java/gax/src/main/java/com/google/api/gax/rpc/ApiCallContext.java

```diff
@@ -266,4 +272,21 @@ public static <T> Key<T> create(String name) {
       return new Key<>(name);
     }
   }
+
+  /**
+   * Validate the Universe Domain to ensure that the user configured Universe Domain and the
+   * Credentials' Universe Domain match. An exception will be raised if there are any issues when
+   * trying to validate (i.e. unable to access the universe domain).
+   *
+   * <p>For both transports, a PermissionDenied exception will be thrown if the universe domain is
+   * invalid
+   *
+   * <p>For gRPC, if the client library is unable to retrieve the Universe Domain, the client
+   * library will attempt to retry with the defined retry bounds. If the retry bounds have been
+   * exceeded, the client library will return an Unavailable exception back.
+   *
+   * <p>For HttpJson, if the client library is unable to retrieve the Universe Domain, the client
+   * library will fail and return an exception.
```

Comment:
These descriptions are good at defining the behavior, but please use the `@throws` Javadoc tag to define the methods' exception behavior, being explicit about the type of exception being thrown in each case.

Also, please be precise with language w.r.t. `throws an exception` vs `returns an exception`

Reply:
Thanks, will update!